doc/sphinx/changes.rst

```diff
@@ -11,9 +11,184 @@ Recent changes
+`docker-coq-action <https://github.com/coq-community/docker-coq-action>`_
+container action (which is applicable to any opam project hosted on GitHub).
+Cyril Cohen, Vincent Laporte, Pierre Roux and Théo Zimmermann
+maintained the `Nix toolbox <https://github.com/coq-community/coq-nix-toolbox>`_.
+The docker-coq-action and the Nix toolbox are used by many Rocq projects for continuous integration.
+
+Ali Caglayan, Emilio Jesús Gallego Arias, Rudi Grinberg and
+Rodolphe Lepigre maintained the
+`Dune build system for OCaml and Coq/Rocq <https://github.com/ocaml/dune/>`_
+used to build Rocq itself and many Rocq projects.
+
+The `opam repository <https://github.com/coq/opam>`_ for Rocq packages has been maintained by
+Guillaume Claret, Guillaume Melquiond, Karl Palmskog, Matthieu Sozeau
+and Enrico Tassi with contributions from many users. The up-to-data list
+of packages is `available on the Rocq website <https://rocq-prover.org/packages>`_.
+
+Rocq 9.0 was made possible thanks to the following reviewers:
+TODO
+
+See the `Rocq Team <https://rocq-prover.org/governance>`_ page for
+more details on Rocq's development teams.
+
+The ?? contributors to the 9.0 version are:
+TODO
+
+The Coq/Rocq community at large helped improve this new version via
+the GitHub issue and pull request system, the coq-club@inria.fr mailing list,
+the `Discourse forum <https://coq.discourse.group/>`_ and the
+`Coq Zulip chat <https://coq.zulipchat.com>`_.
+
+Version 9.0's development spanned 7 months from the release of Coq 8.20.0.
+Pierre-Marie Pédrot and Matthieu Sozeau are the release managers of Rocq 9.0.
+This release is the result of ?? merged PRs, closing ?? issues.
+
+| Nantes, January 2025
+| Pierre-Marie Pédrot and Matthieu Sozeau for the Rocq development team
+
+Porting to The Rocq Prover
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Rocq Prover version 9.0 includes compatibility shims that make it possible
+to invoke it through legacy Coq commands: `coq-tex`, `coq_makefile`, `coqchk`, `coqdoc`, `coqpp`, `coqtop`,
+`coqwc`, `coqc`, `coqdep`, `coqnative`, `coqtimelog2html`, `coqtop.byte`, `coqworkmgr`.
+When using `opam`, this compatibility layer is available through packages `coq-core`,
+`coq-stdlib` and `coq`. In this setting, nothing needs to be changed to the build systems
+of existing projects to compile with `Rocq 9.0` (aliased as `Coq 9.0`).
+
+You should expect warnings that the standard library previously under namespace
+`Coq` has been renamed to `Stdlib`. See `this entry
+<https://rocq-prover.org/doc/v9.0/refman-stdlib/changes.html#changed>`_ from
+the Standard Library's changelog for the suggested workflow to port theories.
+
+There are important changes to consider for building plugins and libraries:
+
+- To be future-proof, projects based on `coq_makefile` can be ported to not rely
+  on the compatibility layer anymore. To do so, one must replace uses of
+  `coq_makefile` with :ref:`rocq makefile <rocq_makefile>`, which will directly
```

Comment:
maybe
```suggestion
  `coq_makefile` with :ref:`rocq makefile`, which will directly
```
(the assymetry with `<rocq_makefile>` is a tad disturbing)

Reply:
Careful! This is a `:ref:`. I am not sure Sphinx automatically converts spaces to underscores.

Reply:
You're right, I misread that, please ignore my comment.